src/components/RequireLogin.tsx

```diff
@@ -64,14 +111,18 @@ const RequireLogin: React.FC<Props> = (props: Props) => {
       }));
       // Lets test auth.
       try {
-        await client.go().get();
-        // Keep this message?
-        console.log('[ketting] Stored credentials were accepted');
-        // Authentication succeeded
-        setAuthenticated(true);
-
-        // End function
-        return;
+        if (props.testEndpoint === null) {
+          console.log('[ketting] Using stored credentials. testEndpoint is disabled.');
+          setAuthenticated(true);
+        } else {
+          await client.go(props.testEndpoint).get();
+          console.log('[ketting] Stored credentials were accepted');
```

Comment:
Would it be worth including the testEndpoint or default endpoint URI that is hit in these logs? Would maybe be a more explicit way to show the user their `testEndpoint` flag worked

Reply:
Good idea!